src/pages-and-resources/teams/Settings.jsx

```diff
@@ -0,0 +1,150 @@
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Button, Form, Icon } from '@edx/paragon';
+import { Add } from '@edx/paragon/icons';
+import { FieldArray } from 'formik';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { v4 as uuid } from 'uuid';
+import * as Yup from 'yup';
+import { setupYupExtensions, useAppSetting } from '../../utils';
+import AppSettingsModal from '../app-settings-modal/AppSettingsModal';
+import messages from './messages';
+import TeamSetEditor, { TeamSetTypes, TeamSizes } from './TeamSetEditor';
+
+setupYupExtensions();
+
+function TeamSettings({
+  intl,
+  onClose,
+}) {
+  const [teamsConfiguration, saveSettings] = useAppSetting('teamsConfiguration');
+  const blankNewTopic = {
+    name: '',
+    description: '',
+    type: 'open',
+    maxTeamSize: TeamSizes.DEFAULT,
+    id: null,
+  };
+
+  const handleSettingsSave = (values) => {
+    // For newly-added teams, fill in an id.
+    const topics = values.topics.map(topic => ({
+      id: topic.id || uuid(),
+      name: topic.name,
+      type: topic.type,
+      description: topic.description,
+      max_team_size: topic.maxTeamSize,
+    }));
+    saveSettings({ topics, max_team_size: values.maxTeamSize });
+  };
+
+  return (
+    <AppSettingsModal
+      appId="teams"
+      title={intl.formatMessage(messages.heading)}
+      enableAppHelp={intl.formatMessage(messages.enableTeamsHelp)}
+      enableAppLabel={intl.formatMessage(messages.enableTeamsLabel)}
+      learnMoreText={intl.formatMessage(messages.enableTeamsLink)}
```

Comment:
Team helper text is not matching with figma.
current design:
<img width="765" alt="Screenshot 2021-09-10 at 2 52 57 PM" src="https://user-images.githubusercontent.com/79941147/132855435-b0cde6f5-e15c-4fde-bf9c-38c1da1fb160.png">

Figma design:
<img width="645" alt="Screenshot 2021-09-10 at 5 47 25 PM" src="https://user-images.githubusercontent.com/79941147/132855475-4572927e-05c8-4544-b73c-b06df57e8adc.png">